<?xml version="1.0"?>
<launch>
    <!--Plane Segmentation-->
    <param name="/camera/ds_size" type="double" value="0.03" />
    <node pkg="sensor_fusion" type="camera_seg_plane" name="lidar_seg_plane" output="screen">
        <remap from="/cloud" to="/zed/point_cloud/cloud_registered" />
        <remap from="/output" to="/zed/ds_cloud" />
    </node>
</launch>
